### PR TITLE
VZ-6629: Update kubectl in rancher and rancher-agent image for release-1.3

### PR DESCRIPTION
**Pull request by bhatpmk**
Updates kubectl in rancher and rancher-agent image for release-1.3

### CODE REVIEW

**Review by agv426 — APPROVED** [review]

File: Dockerfile.dapper [hunk 1]
```diff
@@ -58,9 +58,10 @@ RUN curl -sLf ${KUSTOMIZE_URL} | tar -xzf - -C /usr/bin
 
 RUN yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64/ && \
     yum install -y helm-3.3.4-2.el7.x86_64 && \
-    yum -y install kubectl-1.20.11-4.el7.x86_64 && \
     yum -y install docker-engine-19.03.11.ol-13.el7.x86_64 && \
     yum -y install etcd-3.4.13-1.el7.x86_64 && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64/ && \
```
Comment (lake-of-dreams) [inline Dockerfile.dapper:63]: will we need https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64 still if we add https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64 ? I guess we should be able to remove one of them

File: package/Dockerfile.agent [hunk 2]
```diff
@@ -19,8 +19,9 @@ RUN yum update -y && \
     yum-config-manager --add-repo=http://yum.oracle.com/repo/OracleLinux/OL7/SoftwareCollections/x86_64 && \
     yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64/ && \
     yum install -y rh-git227 curl docker-cli ca-certificates jq iproute vim less bash-completion unzip sysstat acl tar hostname openssl && \
-    yum -y install kubectl-1.20.11-4.el7.x86_64 && \
     yum -y install docker-engine-19.03.11.ol-13.el7.x86_64 && \
+    yum-config-manager --add-repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne15/x86_64/ && \
```
Comment (lake-of-dreams) [inline package/Dockerfile.agent:23]: should probably be added below the other `yum-config-manager` commands for better readability

File: Dockerfile.dapper (hunk 1 above)
Comment (bhatpmk) [reply Dockerfile.dapper:63]: Repo https://yum.oracle.com/repo/OracleLinux/OL7/olcne13/x86_64 is required for helm-3.3.4-2.el7.x86_64 and etcd-3.4.13-1.el7.x86_64

File: package/Dockerfile.agent (hunk 2 above)
Comment (bhatpmk) [reply package/Dockerfile.agent:23]: My intention is to add just before installing the package from that repo.

**Review by lake-of-dreams — APPROVED** [review]

**Pull request merged**